### PR TITLE
Fix issue #178

### PR DESCRIPTION
**Pull request by CharlesVaneenoo**
Hello, 
We fixed the issue #178.
Thanks.

### CODE REVIEW

File: data/plugins/location/tutorial-island/instructions.rb [hunk 1]
```diff
@@ -73,6 +73,14 @@ def self.show_instruction(player)
              'inventory. Your inventory is a list of everything you have in your backpack.'
     end
 
+    # The instruction displayed if the player tries to cut a tree before having the axe.
+    dialogue :try_cut_tree do
+      type :text
+
+      title 'Follow the guide\'s instructions'
+      text 'You cannot cut down this three, you must first follow the guide\'s instructions.'
```
Comment (Major-) [inline data/plugins/location/tutorial-island/instructions.rb]: This says `three` instead of `tree`.

**Comment by Major-** [comment]
Excellent, thanks. There's a typo that I've commented on, but otherwise this looks great - if you could fix the typo I'll merge this.

**Comment by AymericDu** [comment]
Check. Thanks !

**Comment by Major-** [comment]
Thanks!

**Pull request merged**